components/post_view/message_attachments/message_attachment/__snapshots__/message_attachment.test.tsx.snap

```diff
@@ -591,7 +593,6 @@ exports[`components/post_view/MessageAttachment should match value on getFieldsT
       <tr>
         <th
           className="attachment-field__caption"
-          width="50%"
```

Comment:
This is removed since according to typescript, the width attribute does not exist in th elements.

Reply:
Oh yeah, that's some pre-CSS HTML
![Screen Shot 2021-01-20 at 11 47 29 AM](https://user-images.githubusercontent.com/3277310/105207247-49786580-5b15-11eb-980f-c188deff7b04.png)